src/tests/architect_tests/test_planner_builders.py

```diff
@@ -11,6 +11,7 @@
 
 from triage.component import metta
 from triage.component.architect import Planner, builders
+from triage.component.catwalk.db import ensure_db
```

Comment:
We should probably move ensure_db out of catwalk now. We can create an issue for this though, instead of worrying about it in this PR.